internal/api/design/swagger.go

```diff
@@ -9,7 +9,7 @@ var _ = Service("swagger", func() {
 	HTTP(func() {
 		Path("/swagger")
 	})
-	Files("/swagger.json", "internal/api/gen/http/openapi.json", func() {
+	Files("/swagger.json", "openapi.json", func() {
```

Comment:
I think that the second parameter should read: `gen/http/openapi.json`.

That would be the path used to access the embedded filesystem.

Reply:
Ah! That did the trick, thanks!! :relieved: